src/features/events/components/groupedEventsCard.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import moment from 'moment'
+import { FlexColumn } from '../../../components/grid'
+import { media } from '../../../theme/media'
+import constants from '../../../constants'
+import EventListingCard from './eventListingCard'
+
+const DateGroupHeading = styled.h2`
+  margin: 1rem 0;
+
+  ${media.tablet`
+    display: none;
+  `};
+`
+
+const GroupedEventsCards = ({ event, index, events, toLoad }) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There should be no space before '}'](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4658697)